compiler/il/OMRDataTypes_inlines.hpp

```diff
@@ -200,4 +200,48 @@ OMR::DataType::operator>(TR::DataTypes rhs)
    return _type > rhs;		
    }		
 
+TR::DataTypes
+OMR::DataType::createVectorType(TR::DataTypes elementType, TR::VectorLength length)
+   {
+   TR_ASSERT_FATAL(elementType > TR::NoType && elementType <= TR::NumVectorElementTypes,
+                   "Invalid vector element type %d\n", elementType);
+   TR_ASSERT_FATAL(length > TR::NoVectorLength && length <= TR::NumVectorLengths,
+                   "Invalid vector length %d\n", length);
+
+   TR::DataTypes type = (TR::DataTypes)(TR::NumNonVectorTypes + (length-1) * TR::NumVectorElementTypes + elementType - 1);
```

Comment:
Please use `static_cast` rather than the C-style cast.

Reply:
fixed